qucs/qucs/mouseactions.cpp

```diff
@@ -319,6 +356,8 @@ void MouseActions::MMoveElement(Schematic *Doc, QMouseEvent *Event)
  */
 void MouseActions::MMoveWire2(Schematic *Doc, QMouseEvent *Event)
 {
+//ALYS - i saved old code inside #if 0 block
```

Comment:
i can delete this PR, remove code if #if 0 blocks and create new PR.
is it good?
ps. i see in code some old commented lines, do not know who is author. May be delete them too?